src/openqaoa-braket/backends/gates_braket.py

```diff
@@ -0,0 +1,94 @@
+from typing import Callable
+
+from braket.circuits import gates, Circuit
+
+from openqaoa.qaoa_components.ansatz_constructor.rotationangle import RotationAngle
+import openqaoa.qaoa_components.ansatz_constructor.gates as gates_core
+
+class BraketGateApplicator(gates_core.GateApplicator):
+
+    BRAKET_OQ_GATE_MAPPER = {
```

Comment:
So, I guess a general design choice could be to replace all this with ENUM... it may be worth considering for a future improvement

Reply:
Should be possible. Need to test how we can do this.